src/java.base/share/classes/jdk/internal/foreign/abi/ABIDescriptor.java

```diff
@@ -43,9 +43,14 @@ public class ABIDescriptor {
     final VMStorage targetAddrStorage;
     final VMStorage retBufAddrStorage;
 
-    public ABIDescriptor(Architecture arch, VMStorage[][] inputStorage, VMStorage[][] outputStorage,
-                         VMStorage[][] volatileStorage, int stackAlignment, int shadowSpace,
-                         VMStorage targetAddrStorage, VMStorage retBufAddrStorage) {
+    public ABIDescriptor(Architecture arch,
+                         VMStorage[][] inputStorage,
+                         VMStorage[][] outputStorage,
+                         VMStorage[][] volatileStorage,
+                         int stackAlignment,
+                         int shadowSpace,
+                         VMStorage targetAddrStorage,
+                         VMStorage retBufAddrStorage) {
```

Comment:
Please move the line splitting changes to a separate PR. I think more discussion is needed on that. But we can move forward with some of the changes in this PR I think.

Reply:
I will hold back the other splittings. Can we accept the ones here to move forward?

Reply:
No. We don't have consensus on whether these changes are good or bad, so we should not just move forward with them. We're not in a hurry.